Content.Shared/DeltaV/SpaceFerret/CanBackflipComponent.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Content.Shared.Actions;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization;
+
+namespace Content.Shared.DeltaV.SpaceFerret;
+
+[RegisterComponent]
+public sealed partial class CanBackflipComponent : Component
+{
+    [DataField]
+    public EntProtoId BackflipAction = "ActionBackflip";
+
+    [DataField]
+    public EntityUid? BackflipActionEntity;
+
+    [DataField]
+    public string ClappaSfx = "";
+}
+
+public sealed partial class BackflipActionEvent : InstantActionEvent
+{
+}
+
+[Serializable] [NetSerializable]
+public sealed class DoABackFlipEvent(NetEntity actioner, string sfxSource) : EntityEventArgs
```

Comment:
from before remove sfxsource and make it a record struct

Reply:
class is a heal allocated object struct isnt iirc, just becomes the same cost as the NetEntity itself

could be entirely wrong though